resources/views/platform/components/game/devbox-claim-management.blade.php

```diff
@@ -96,6 +102,20 @@ function dropClaim() {
     return confirm(message);
 }
 
+function reviewClaim() {
+    const gameTitle = "{!! html_entity_decode($gameTitle) !!}";
+
+    const message = 'Are you sure you want to change the claim status for ' + gameTitle + ' to In Review?';
+    return confirm(message);
+}
+
+function activateClaim() {
+    const gameTitle = "{!! html_entity_decode($gameTitle) !!}";
+
+    const message = 'Are you sure you want to change the claim status for ' + gameTitle + ' to Active?';
```

Comment:
nit: This message is indeed technically correct, but my brain twisted into a knot when reading it. I think it's because the button says "Complete", the status is currently "In Review", and we're initiating a transition to "Active", from which the claim will be completed imminently.

I'm wondering if the user-facing message here should just be something like:
* Are you sure you want to set this claim to no longer In Review?

Reply:
I kind of struggled with the wording here too. The user is completing the review, not the claim. After the review is completed, the lock is removed and the Jr. Dev is allowed to complete the claim.

Reply:
It's fine to stay as-is. I do think this could be a point of confusion, but the user will probably only get confused the first time they use the control, if at all.